Dockerfile-ppc64le

```diff
@@ -0,0 +1,44 @@
+ARG quay_expiration=never
+ARG release_tag=0.0.0
+
+# golang:1.16 image created 2021-06-24T00:31:06.02014601Z
+FROM docker.io/golang AS builder
```

Comment:
Please add an explicit tag for the version. If 1.17 is available for ppc, use that.